esp/esp/tagdict/__init__.py

```diff
@@ -136,6 +136,7 @@
     'switch_time_program_attendance': (False, 'At what time should the student onsite webapp use program attendance if available (instead of enrollment) to determine if a class is full? If blank, program attendance numbers will not be used. Format: HH:MM where HH is in 24 hour time.', None, 'learn', True),
     'switch_lag_class_attendance': (False, 'How many minutes into a class should the student onsite webapp use class attendance numbers if available (instead of enrollment or program attendance) to determine if a class is full? If blank, class attendance numbers will not be used.', None, 'learn', True),
     'student_lottery_group_max': (False, 'What is the maximum number of students that can be in the same lottery group? (set to 1 to not allow groups)', '4', 'learn', True),
+    'survey_isstep': (True, 'Should the student and teacher surveys be shown as steps in student and teacher registration respectively once the event has started?', False, 'manage', True),
```

Comment:
Why is the survey_isstep a program tag while webapp_isstep is a global tag? I think these isstep tags should be consistent, otherwise it makes them hard to find as an admin. I think as long as we import tags when we import a program, I would probably prefer both to be program tags.

Reply:
Looking at the code changes I don't have any clue as to how to do this, but ideally even with the "survey_isstep", the step should only show up if a student/teacher survey exists for the program. For example if "isstep" is true and there's actually only a teacher survey, it would be not great for the student survey link to show up in student registration and have this error pop up:
![image](https://user-images.githubusercontent.com/25392075/88609134-6134cc80-d051-11ea-8850-c089f51771d9.png)

Reply:
The step in student registration for "surveys" shows up smack in the middle of the list. I think it should probably be the last item. Could we update the seq for the module to make it lower on the list?